test/cases/api.sh

```diff
@@ -348,7 +348,19 @@ function installClientAzure() {
   $AZURE_CMD version
 }
 
+source /etc/os-release
+
+if [[ "$CI" == true ]]; then
```

Comment:
Uh since `set -u` is set it's going to fail outside of CI cause that variable will not be set, you'd need something like `CI="${CI:-false}"` for it to work :)

Reply:
Uups, that makes sense